CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Bob versions changelog
 - Internode auth works properly with nodes on same ip (#548)
 - Fix response in delete request (#558)
 - Fixed panic in brt because of duplicate long arg name (#563)
+- Compare vdiskid first (#594)
```

Comment:
This line should be in `Changed` section, because this is not a bug

Reply:
Fixed: [edb8a8b](https://github.com/qoollo/bob/pull/614/commits/edb8a8b5fc9e908e22950e55353e9d229e0ec227)